content/accessibility/designer-toolkit.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Designer toolkit
+---
+
+Whether you’re designing a new user experience or just one component, make sure you’re addressing the considerations from the respective list.
+## Accessibility checklists
+### Component creation
+Check out a11yportal's [custom component creation checklist for UX designers](https://www.a11yportal.com/guidelines/fundamentals/checklists.html#custom-control-checklist-ux).
+### UX Flow
```

Comment:
I think we need a sentence here to explain what this list is for? When do you use it? Can we spell it out instead of saying "UX"?